src/aiperf/ui/dashboard/realtime_metrics_dashboard.py

```
@@ -34,7 +34,7 @@ class RealtimeMetricsTable(Widget):
     }
     """
 
-    STATS_FIELDS = ["avg", "min", "max", "p99", "p90", "p50", "std"]
+    STATS_FIELDS = ["avg", "min", "max", "p99", "p90", "p50", "adj_p95", "adj_p99", "std"]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)